src/OpenFeature/Providers/Memory/Flag.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections.Generic;
+using OpenFeature.Constant;
+using OpenFeature.Error;
+using OpenFeature.Model;
+
```

Comment:
```suggestion

#nullable enable
```

Let's force the nullable validations/checks in the new class files so it would be easier later on to enable for the entire project.

Reply:
great idea.

Reply:
This IMMEDIATELY helped me find at least 2 possible null pointer errors. Wow.